src/app/components/navigation/navigation.html

```diff
@@ -19,7 +19,7 @@
 
     <md-list ng-show="$ctrl.currentUser">
         <md-subheader class="md-no-sticky">Admin</md-subheader>
-        <md-list-item ng-click="$ctrl.$state.go('expeditions.list')">
+        <md-list-item ng-click="$ctrl.$state.go('expeditions.list')" ng-show="$ctrl.currentUser.userId === $ctrl.currentProject.user.userId">
```

Comment:
this should be visible for any logged in user, not just the project admin

Reply:
This change doesn't do that. It only makes it visible if you are a project admin.

Reply:
You'll need to make another api call to fetch the project members and see if the currentUser is in that list